node/chainSimulator/components/statusCoreComponents.go

```diff
@@ -24,25 +19,41 @@ type statusCoreComponentsHolder struct {
 }
 
 // CreateStatusCoreComponentsHolder will create a new instance of factory.StatusCoreComponentsHolder
-func CreateStatusCoreComponentsHolder(cfg config.Config, coreComponents factory.CoreComponentsHolder) (factory.StatusCoreComponentsHolder, error) {
+func CreateStatusCoreComponentsHolder(configs config.Configs, coreComponents factory.CoreComponentsHolder) (factory.StatusCoreComponentsHandler, error) {
```

Comment:
Please rename to `CreateStatusCoreComponents` and remove the inner closeHandler declaration & usage

Reply:
renamed